web/src/js/utils/local-user-data-mgr.js

```diff
@@ -357,3 +359,30 @@ export const setUserDismissedSurvey2022 = () => {
 export const hasUserDismissedSurvey2022 = () => {
   return localStorageMgr.getItem('tab.user.dismissedUserSurvey2022') === 'true'
 }
+
+export const getLoggedOutMessage = () => {
+  const installTime = getBrowserExtensionInstallTime()
+
+  if (!installTime) {
+    return LOGGED_OUT_MESSAGE_TYPE.NONE
+  }
```

Comment:
Let's edit this logic (sorry for the unclear spec): consider a user "new" if there is an install time and it's recent; otherwise, assume the user is old. It's unlikely an existing user who's logged out will have stuff set in local storage, so the current logic would probably rarely show to existing users.